### DISCUSSION TITLE
Information :)

### DISCUSSION THREAD

**Discussion by l3str4nge** [TheAlgorithms/Python#8580] [Team Posts]
Hi I want to mention that I've just changed my username but I am still old mateuszz00000 ;)

**Comment by itsvinayak** [comment]
nice, I like L3str4nge more than mateuszz00000

**Comment by putul03** [comment]
Hi @l3str4nge @itsvinayak  and the team! I understand you have a lot on your plate with many open pull requests. If you could take a moment to review my PR - Added Python Implementation of Suffix Arrays and LCP Arrays #12171, it would be greatly appreciated. I'm excited to contribute to the project and welcome any feedback you may have. Thank you!
